ADotNet/Models/Pipelines/GithubPipelines/DotNets/BuildJob.cs

```diff
@@ -12,12 +12,21 @@ namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets
 {
     public class BuildJob
     {
+        private string _runsOn;
```

Comment:
Let's not have underscore for private class members.

here's the coding standard: https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Classes.md#411-referencing
here's for reference: https://github.com/hassanhabib/CSharpCodingStandard/issues/6

Reply:
@hassanhabib , out of curiosity, why the need for a separate coding convention if there is a common one for c# already?
https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case
It actually suggest using underscores and prefixes for private and internal fields (s_ for static, t_ for thread static).